### ISSUE TITLE
Dice Score Evaluation

### ISSUE THREAD

**Issue by JoTisch** [mabulnaga/multimorph#1]
First of all, congratulations on the excellent work. Hope you had a great time at CVPR!

I’ve been reading through your paper and had a couple of questions regarding the evaluation procedure, specifically the way atlas-based segmentation and Dice score computation are handled. In your paper, you mention:

> Each atlas is estimated using half the subgroup (m/2 images). We randomly sample (m/2) segmentation label maps to generate the atlas segmentation mask, which is then transferred to the remaining (m/2) images. Segmentation quality is assessed using the Dice score.

A bit earlier, you also state:

> We compute the Dice score to assess how well the atlas aligns with warped subject scans.

I would really appreciate it if you could help clarify the following:

1. How is the atlas segmentation propagated to the remaining images? Are the remaining images also warped (to obtain deformation fields), but later excluded for the atlas construction? E.g., in `build_atlas_inference.py`
```
# predict deformation for all images 
predicted_warp = model(image)
warped_group = warp_layer(image, predicted_warp)
# compute the atlas but exclude half of the dataset.  
num_exclude = num_imgs / 2
atlas = torch.mean(warped_group[:, :-num_exclude], dim=1, keepdim=True)
```
2. How is the Dice calculated? Between the atlas segmentation and the warped subject segmentation? or between the warped subject segmentation and its GT? 

Thanks in advance, and looking forward to your insights!
